app/models/notification.rb

```diff
@@ -47,14 +47,14 @@ def send_new_comment_notifications(comment)
       return if comment.commentable_type == "PodcastEpisode"
```

Comment:
On a side note. I was not able to find where this method is being called - **send_new_comment_notifications**
Could it be that it is not needed and is better deleted?

Reply:
Yeah, I did a search myself and could not find it used anywhere. For the sake of simplicity, I say lets delete it.

Reply:
Added a commit to this PR that removes the method discussed.